### PR TITLE
captcha hotfix

### PR DESCRIPTION
**Pull request by OmerGery**

### CODE REVIEW

File: examples/client/Locomotion/src/pages/Profile/Phone.js [hunk 1]
```diff
@@ -49,10 +47,9 @@ const Phone = ({ navigation }) => {
   }, []);
 
 
-  const onVerifyCaptcha = async (token) => {
-    Mixpanel.setEvent('Captcha Verified successfully', { token });
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/Profile/Phone.js:53]: token is a reserved word in mixpanel, caused this log to never be sent
https://docs.mixpanel.com/docs/data-structure/events-and-properties

File: examples/client/Locomotion/src/pages/Profile/Phone.js [hunk 2]
```diff
@@ -80,6 +77,7 @@ const Phone = ({ navigation }) => {
       if (!isDevSettingOn()) {
         await AppSettings.destroy();
       }
+      await Auth.updateCaptchaToken(captchaToken);
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/Profile/Phone.js:80]: https://github.com/Autofleet/locomotion/pull/833/files#diff-c05e09a619ce22ac8b2863115ae42f257ce102e5c7aa326a1f36740dd7f1b95fR78 is deleting the token from the storage. moved it to be after the destroy

**Review by atlowChemi — APPROVED** [review]

**Review by barspielberg — APPROVED** [review]

File: examples/client/Locomotion/src/pages/Profile/Phone.js (hunk 1 above)
Comment (sephina) [reply examples/client/Locomotion/src/pages/Profile/Phone.js:53]: what are you doing awake on 7 am?

**Review by sephina — APPROVED** [review]

File: examples/client/Locomotion/src/pages/Profile/Phone.js (hunk 1 above)
Comment (ariel-weiss) [reply examples/client/Locomotion/src/pages/Profile/Phone.js:53]: oww edgy

that's why we need typescript everywhere

Comment (OmerGery) [reply examples/client/Locomotion/src/pages/Profile/Phone.js:53]: Dreaming of captcha issues

**Pull request merged**